<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-54GBLK4C" failures="1" tests="5" name="Personal Loan" time="0.11" errors="0" timestamp="2024-06-23T23:59:08 IST">
  <testcase classname="test.Day1" name="Demo" time="0.017">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Day1.Demo(Day1.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="test.Day2" name="ploan" time="0.003"/>
  <testcase classname="test.Day4" name="MobileLoginHomeLoan" time="0.004"/>
  <testcase classname="test.Day4" name="webLoginHomeLoan" time="0.009"/>
  <testcase classname="test.Day4" name="APILoginHomeLoan" time="0.004"/>
  <testcase classname="test.Day1" name="secondTest" time="0.0">
    <ignored/>
  </testcase> <!-- secondTest -->
</testsuite> <!-- Personal Loan -->
